src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodec.java

```diff
@@ -388,5 +407,16 @@ JsonNode apply(ObjectMapper objectMapper, DataType fromCQLType, CqlT value)
     static <CqlT> ToJSON<CqlT> unsafeNodeFactory(Function<CqlT, JsonNode> nodeFactoryMethod) {
       return (objectMapper, fromCQLType, value) -> nodeFactoryMethod.apply(value);
     }
+
+    /**
+     * Returns an instance that will translate given CQL value using plain {@code toString()},
+     * constructing a {@link JsonNode} of type {@link com.fasterxml.jackson.databind.node.TextNode}.
+     *
+     * <p>See usage in the {@link JSONCodecRegistry}
+     */
+    static <CqlT> ToJSON<CqlT> toJSONUsingToString() {
```

Comment:
Feel strange to the method name at the first glance.
But understand what it does after reading more info.
Just a suggestion, probably not good, method name can be something like `toJSONFromCqlPlaneString`?

Reply:
Let me improve javadoc a bit: pertinent part to me is that it is `value.toString()` that gets called, as `String.valueOf()` is defined as:

```
    public static String valueOf(Object obj) {
        return (obj == null) ? "null" : obj.toString();
    }
```

and this is why I'd like to have "toString" in name.